bin/build-dev

```diff
@@ -3,9 +3,12 @@
 # DOC: Build a new development docker image and optionally push to Docker Hub if PUSH_IMAGE is set to anything.
 
 source bin/lib.sh
+export DOCKER_BUILDKIT=1
+
+PLATFORM="${PLATFORM:-"linux/amd64"}"
```

Comment:
Can you add comment explaining what this line is doing? Is that bash string manipulation/substring?

Reply:
This just sets a default for the variable